### PR TITLE
Metrics: gnt sent/received

### PR DESCRIPTION
**Pull request by jiivan**
fixes: #613

### CODE REVIEW

**Review by nieznanysprawiciel — APPROVED** [review]

File: core/payment/src/service.rs [hunk 1]
```diff
@@ -491,8 +493,11 @@ mod public {
             return Err(SendError::BadRequest("Invalid payer ID".to_owned()));
         }
 
+        let driver = payment.payment_platform.clone();
```
Comment (Wiezzel) [inline core/payment/src/service.rs:496]: Driver is not the same thing as platform. It doesn't matter for now, but it's a semantic error.

File: core/metrics/src/lib.rs [hunk 2]
```diff
@@ -3,3 +3,14 @@ mod metrics;
 mod service;
 
 pub use service::MetricsService;
+
+pub mod utils {
+    const CRYPTOCURRENCY_PRECISION: u64 = 1000000000;
```
Comment (Wiezzel) [inline core/metrics/src/lib.rs:8]: That's of course problematic if we allow other currencies than NGNT.

File: core/payment/src/service.rs (hunk 1 above)
Comment (tworec) [reply core/payment/src/service.rs:496]: fixed via #683

File: core/metrics/src/lib.rs (hunk 2 above)
Comment (tworec) [reply core/metrics/src/lib.rs:8]: no such plans afaik

**Pull request merged**